UPGRADE.md

```diff
@@ -33,16 +33,21 @@ With the release of Kuma 2.6.0, we've made some changes to the implementation of
 
 To ensure a smooth transition to Kuma 2.6.0, carefully review your existing configuration files and make necessary adjustments related to denied request responses and RBAC-related Envoy stats.
 
+### Make format SI valid for bandwidth in MeshFaultInjection policy
```

Comment:
Nitpick: 

```suggestion
### Make SI format valid for bandwidth in MeshFaultInjection policy
```

Reply:
This part is not directly related to task at this PR. What I've done was shifted this section above. 
I'll try to apply the suggestions, but want the original author to be informed. @lahabana